irs/src/components/Login/Login.js

```diff
@@ -5,27 +5,27 @@ import { Button, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
 class Login extends React.Component {
   state = {
     credentials: {
-      username: "",
-      password: "",
-      authLevel: "",
-      organization: "",
+      username: '',
+      password: '',
+      authLevel: '',
+      organization: ''
     }
   };
   render() {
     return (
-        <Form>
-            <FormGroup>
-                <Label for="exampleEmail">Email</Label>
-                <Input type="email" name="email" id="exampleEmail" placeholder="with a placeholder" />
-            </FormGroup>
-            <FormGroup>
-                <Label for="examplePassword">Password</Label>
-                <Input type="password" name="password" id="examplePassword" placeholder="password placeholder" />
-            </FormGroup>            
-            <Button>Submit</Button>
-        </Form>
+      <Form>
+        <FormGroup>
+          <Label for="exampleEmail">Email</Label>
+          <Input type="email" name="email" id="exampleEmail" placeholder="with a placeholder" />
```

Comment:
make sure to add the `required` attribute on each input field so the user can't submit an empty string